packages/core/src/types/v3/people/base-cast-crew-credits.ts

```diff
@@ -0,0 +1,33 @@
+export interface PeoplePersonCredit {
+    adult: boolean;
+    backdrop_path: string;
+    genre_ids: number[];
+    id: number;
+    original_language: string;
+    overview: string;
+    popularity: number;
+    poster_path: string;
+    vote_average: number;
+    vote_count: number;
+    credit_id: string;
+
+}
+ interface PeopleCastAndCrewGetCombinedGetTvCredit extends PeoplePersonCredit {
+    original_title: string;
+    release_date: string;
+    title: string;
+    video: boolean;
+}
+interface PeopleCastCreditSpecificFields{
+    character: string;
+}
+ type PeopleCrewCreditSpecificFields = { job: string; department: string; }
```

Comment:
interface 🙂

Reply:
Ok gotcha there. Thought you might have wanted me to change that other one to an interface because it was used in an interface and leave this one a type because it was used in a type.

Reply:
Ah ye idk i generally use interfaces, unless a new type is created which only extends other types. Im not saying its the right way, just my own little rule :P 

```
interface Foo {
     bar: string
}

interface Bar {
     foo: string
}

interface ExtendedFoo extends Foo {
    extendedFoo: string
}

// Because otherwise you end up with extends and an empty object, which seems ugly to me 😂 
type FooBar = Foo & Bar;
```

Reply:
Yeah I normally try not have those empty interfaces in my own projects because it angers the linter and means I might overlook more  comparatively  serious when  scroll further on the issues/problems page in my ide

Would you be ok with me sliding one extra, unrelated to these issues, change in here  because SearchMoviesParams type from the search endpoint  has its "year" field taking a number, but now it takes a string according to the api website. I could do a new pr if needed

Reply:
Nope feel free to update as many endpoints as you want! 😁